CHANGELOG.md

```diff
@@ -6,13 +6,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.1.0] - 2024-03-18
+
+### Added
+- Added a new endpoint for sending updates for single field values. (DR-2775)
+- Exposed delayed jobs through delayed job web. (DR-2830)
+
 ## [2.0.1] - 2024-01-16
 
 ### Fixed
 - Fixed additional issue causing suppressed captures to be pulled in for indexing. (DR-2690)
 
 ### Added
 - Added one-off script to cleanup empty collections and containers. (DR-2557)
+- Added firstIndexed_dt and dateIndexed_dt to repoapi solr docs. (DR-2789)
```

Comment:
Actually...it looks like this ticket has not been released yet:
https://jira.nypl.org/browse/DR-2789
<img width="1480" alt="Screenshot 2024-03-18 at 12 25 50 PM" src="https://github.com/NYPL/fedora_ingest_rails/assets/4614468/8dbd41e0-3607-4306-872d-73911b69e806">

Should this be added to the Unreleased section?

Reply:
(since [DR-2789](https://jira.nypl.org/browse/DR-2789) involves both MMS and Fedora Ingest, here the PR for fedora ingest: #131.  This PR was merged into QA, but is not included in this PR, and is not in production)

Reply:
Oh dang. Well-spotted, @kayleiburke .